src/main/java/org/reaktivity/nukleus/tls/internal/stream/ServerStreamFactory.java

```diff
@@ -974,17 +1016,16 @@ private void updateNetworkWindow(
         private void setNetworkThrottle(
             MessageConsumer newNetworkThrottle)
         {
-            if (networkReplyResetAfterHandshake)
-            {
-                doReset(newNetworkThrottle, networkReplyId);
-            }
-            else
-            {
-                router.setThrottle(networkReplyName, networkReplyId, newNetworkThrottle);
+            router.setThrottle(networkReplyName, networkReplyId, newNetworkThrottle);
 
-                // TODO: improve heuristic to account for TLS protocol framing bytes
-                doWindow(newNetworkThrottle, networkReplyId, outNetworkWindowBytes * 80 / 100, outNetworkWindowFrames);
-            }
+            // TODO: improve heuristic to account for TLS protocol framing bytes
+            doWindow(newNetworkThrottle, networkReplyId, outNetworkWindowBytes * 80 / 100, outNetworkWindowFrames);
+        }
+
+        private void setApplicationReplyCleaner(
+            Runnable applicationReplyCleaner)
+        {
+            applicationInCleanerConsumer.accept(applicationReplyCleaner);
```

Comment:
Rename `applicationReplyCleaner` to `networkReplyDoneHandler`.
Rename `applicationInCleaner` to `networkReplyDoneHandlerConsumer`.
Rename `setApplicationReplyCleaner` to `setNetworkReplyDoneHandler`.